onnxruntime/contrib_ops/cuda/collective/custom_reduce_impl.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2022-2024, NVIDIA CORPORATION.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/providers/cuda/cuda_kernel.h"
+#include <cuda_fp16.h>
+
+namespace ort_trtllm {
```

Comment:
>amespace ort_trtllm { [](http://example.com/codeflow?start=1&length=21)

Would this not conflict? Should this be wrapped into onnxruntime namespace?

Reply:
This name should be unique